clang/test/Driver/cxx23-modules-print-library-module-manifest-path.cpp

```diff
@@ -0,0 +1,15 @@
+// Test that -print-library-module-manifest-path finds the correct file.
+
+// RUN: %clang -print-library-module-manifest-path \
+// RUN:     -stdlib=libc++ \
+// RUN:     --sysroot=%S/Inputs/cxx23_modules \
+// RUN:     --target=x86_64-linux-gnu 2>&1 \
+// RUN:   | FileCheck --check-prefix=CHECK-LIBCXX %s
+// CHECK-LIBCXX: {{.*}}/Inputs/cxx23_modules/usr/lib/x86_64-linux-gnu/modules.json
```

Comment:
Let's try to avoid use `Inputs` pattern nowadays. We can use `split-file` to multiple files into a single lit test. You can find the use example by searching `split-file` in `clang/test/Modules`